chia/plotting/plot_info.py

```diff
@@ -0,0 +1,21 @@
+import logging,sys
+
+from chia.plotting.plot_tools import load_plot
+
+log = logging.getLogger(__name__)
+
+
+def plot_info(file: str):
+    (plot_info,error) = load_plot(file)
```

Comment:
I think you might get a test warnings also due to formatting issues such as blanks after commas, as in `a, b = f()`.
Also below, e.g. in `return   {},"use -f`.
In any case, right above that line I think it wants you to do `is None` instead `==None`.

----
Btw. if you want to spare some characters, which in this case is more of a style question, you can use some idioms to turn this into
```python
    plot_info, error = load_plot(file)  # brackets not necessary
    if error:  # !="" not necessary
        print(error)
        sys.exit(1)
    else:
        print("\n".join([  # newlines instead of more prints
            f"path: {file}", 
            f"k: {plot_info.prover.get_size()}",
            f"id: {pr.get_id().hex()}",
            f"farmer public key: {plot_info.plot_public_key}",
            f"pool public key: {plot_info.pool_public_key}",
        ])
        sys.exit(0)
```